content/_releases/22/38.md

```diff
@@ -0,0 +1,161 @@
+
+We changed the wording to highlight that this address to be used **only one-time** to
+send a representative sample which you can use to configure subsequent steps in your
+integration flow.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2238/RN2238-webhook-send-sample-request.png" title="WebHook step: send sample request" %}
+
+### REST-API-V2 component POST and PUT body type
+
+While configuring the REST-API-v2 component step in integration flows to `POST` or
+`PUT` JSON body, we use `application/json` content type but forget to change the
+input field from [Integrator to JSONata mode](/guides/mapping-data#jsonata-mode).
+When the configuration is saved, the HTTP call body is saved with an incorrect
+format resulting in unexpected or erroneous outcome during the flow execution.
+
+To improve the user experience and prevent an accidental JSON data structure pasting
+into the Body field while in an Integrator mode, we changed the User Interface
+behaviour to default into JSONata mode when an `application/json` content type is
+selected for the `POST` and `PUT` HTTP calls.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2238/RN2238-rest-api-v2-jsonata.png" title="REST-API-V2 step: JSONata mode for POST/PUT" %}
+
+### Run-time environment variables
+
+During the integration step execution, components exchange information with the
+platform services and database entities using already defined run-time environment
+variables. We are constantly working to standardise and add new environment variables
+to streamline the information exchange. The following environment were added to
+the run-time:
+
+*   `WEBHOOK_URI` - This environment variable will be available to each step in the flow which exchanges the data with the webhook service of the platform, eg. [WebHook](/components/webhook/) or [Flow-linking](/components/flow-linking/). This environment variable from the Tenant attributes `webhooks_domain` or from the HELM3 environment variable `WEBHOOKS_BASE_URI` if the Tenant record is missing.
+
+*   `ELASTICIO_CONTRACT_ID` - This environment variable will be available during the run-time of every execution including:
+    *   Verify credentials
+    *   Select model
+    *   Dynamic metadata
+    *   Generated samples and flow execution
+
+## Updates
+
+### Node.js Sailor version `2.7.0`
+
+A new version of Nod.JS Sailor library `2.7.0` is released. This update lays a
+preparatory groundwork towards the message persistence feature.
+
+### Java Sailor version `3.4.0`
+
+A new version of Java Sailor library `2.7.0` is released. This update lays a
+preparatory groundwork towards the message persistence feature.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Message persistence configurations
```

Comment:
Please remove - we do not yet know how this might affect a tenant. Dev team is making some load tests / simulations